tests/test_api.py

```diff
@@ -125,3 +125,54 @@ def test_record_performance_file_name(self):
         perf_name = __file__.replace('.py', '.file_name.perf.yml')
         with self.record_performance(file_name=perf_name):
             caches['default'].get('foo')
+
+
+class FileNameTests(TestCase):
+
+    default_perf_name = __file__.replace('.py', '.perf.yml')
+    custom_perf_name = 'custom.perf.yml'
+    custom_path_perf_name = os.path.join(
+        os.path.dirname(__file__),
+        'performance_tests',
+        os.path.basename(__file__).replace('.py', '.perf.yml')
+    )
+
+    def setUp(self):
+        super(FileNameTests, self).setUp()
+        self.ensure_no_file()
+
+    def tearDown(self):
+        self.ensure_no_file()
+        super(FileNameTests, self).tearDown()
+
+    def ensure_no_file(self):
+        for name in (self.custom_perf_name, self.custom_path_perf_name):
+            try:
+                os.unlink(name)
+            except OSError as exc:
+                if exc.errno != errno.ENOENT:
+                    raise
+
+        try:
+            os.rmdir(os.path.dirname(self.custom_path_perf_name))
+        except OSError as exc:
+            if exc.errno != errno.ENOENT:
+                raise
+
+    def test_default_filename(self):
```

Comment:
this test writes to the default file and you didn't commit the change:

```
+RecordPathTests.test_default_filename:
+- cache|get: foo
```

But it's also kind of a useless test because it's asserting that a *committed file* (`test_api.perf.yml`) exists, so I'm just removing it.